package_policies/R7.md

```diff
@@ -0,0 +1,16 @@
+**R7.** It is recommended that each package should have a README or README.md, a SUPPORT or SUPPORT.md, a LICENSE or LICENSE.md, and a CHANGELOG or CHANGELOG.md file in its top directory.
+The README file should contain at least the following information:
+
+- a **brief** description of the package
+
+- information on how to install the package or a link to a file (e.g. INSTALL, INSTALL.md or similar) or a website with the installation information
+
+- a link to the package webpage (if there is one)
+
+The LICENSE file should contain the full text of the license. If the full text can be found in a different directory or via a link to a website it is sufficient to provide a link to the full text in the file.
+
+The SUPPORT file should contain the contact information that is required by community policy **M5** to be able to get help.
+
+The CHANGELOG file should contain important changes or a link to a file or webpage with this information, but not each individual commit.
```

Comment:
`but not each individual commit` -> `but not a list of all individual commit messages`

Reply:
or:
 `but not each individual commit message`